pkg/didcomm/crypto/legacy/authcrypt/authcrypt_test.go

```diff
@@ -418,9 +454,12 @@ func decryptComponentFailureTest(
 
 	w, s := newWallet(t)
 	err := persistKey(base58.Encode(recKey.Pub), base58.Encode(recKey.Priv), s)
-	require.NoError(t, err)
+	if err != nil {
```

Comment:
no need for if condition here... use `require` directly

Reply:
oh so this is part of a function that can be called for Successful and Failing TCs... that's ok then